node/impl/full/eth.go

```diff
@@ -1571,6 +1571,13 @@ func newEthTxFromSignedMessage(ctx context.Context, smsg *types.SignedMessage, s
 		if err != nil {
 			return ethtypes.EthTx{}, err
```

Comment:
Nit: Can you please also include an error message here? It's _totally_ not related to this PR, but it's been frustrating not knowing where errors have been originating from in the Eth API (so I'm trying to slowly add them).
```suggestion
			return ethtypes.EthTx{}, xerrors.Errorf("failed to calculate hash for ethTx: %w", err)

```

Reply:
Absolutely ❤️